run_unittests.py

```diff
@@ -1144,6 +1153,7 @@ def test_vim_syntax_highlighting(self):
             defined = set([a.strip() for a in res.group().split('\\')][1:])
             self.assertEqual(defined, set(chain(interp.funcs.keys(), interp.builtin.keys())))
 
+    @unittest.skipIf(is_pull(), 'Skipping because this is a pull request')
```

Comment:
I'm disabling this test for pull requests since it will always fail when adding new functions until they are merged to [language-meson](https://github.com/TingPing/language-meson). Local tests and those running against master should still enforce this however.